src/main/java/io/nats/client/impl/NatsDispatcher.java

```diff
@@ -27,26 +27,28 @@
 
 class NatsDispatcher extends NatsConsumer implements Dispatcher, Runnable {
 
-    private MessageQueue incoming;
-    private MessageHandler defaultHandler;
+    protected final MessageQueue incoming;
+    protected final MessageHandler defaultHandler;
```

Comment:
protected since this will be subclasses. final because Intellij says so, probably a good practice